### ISSUE TITLE
SDNObsMetadata returning an empty window

### ISSUE THREAD

**Issue by ctroupin** [gher-ulg/DIVAnd.jl#17]
When running the notebook [09-ODV-data-import.ipynb](https://github.com/gher-ulg/Diva-Workshops/blob/Julia-0.7/notebooks/09-ODV-data-import.ipynb), the line
```julia
SDNObsMetadata(obsid[10])
```
opens a windows but it only contains this text:     
"*The resource you are looking for has been removed, had its name changed, or is temporarily unavailable.*"

Maybe related to a change of URL for SDN?

**Comment by Alexander-Barth** [comment]
Thanks Charles.
I reported it here: https://github.com/JuliaWeb/HTTP.jl/issues/323

**Comment by Alexander-Barth** [comment]
Fixed now in HTTP.jl.

**Comment by ctroupin** [comment]
Ok let's test and close if it works.

**Comment by Alexander-Barth** [comment]
yes, releases in HTTP.jl seem to be on a ~ weekly basis, so it should not take too long to have this fix in a released version.
